addons/mail/static/tests/new/messaging_menu/notification_group_tests.js

```diff
@@ -0,0 +1,334 @@
+import { getFixture, patchWithCleanup } from "@web/../tests/helpers/utils";
+
+let target;
+
+QUnit.module("notification group", {
+    async beforeEach() {
+        target = getFixture();
+    },
+});
+QUnit.test("notification group basic layout", async function (assert) {
+    const pyEnv = await startServer();
+    const mailChannelId1 = pyEnv["mail.channel"].create({});
+    const mailMessageId1 = pyEnv["mail.message"].create({
+        message_type: "email",
+        model: "mail.channel",
+        res_id: mailChannelId1,
+        res_model_name: "Channel",
+    });
+    pyEnv["mail.notification"].create([
+        {
+            mail_message_id: mailMessageId1,
+            notification_status: "exception",
+            notification_type: "email",
+        },
+        {
+            mail_message_id: mailMessageId1,
+            notification_status: "exception",
+            notification_type: "email",
+        },
+    ]);
+    const { click } = await start();
+    await click(".o_menu_systray i[aria-label='Messages']");
+    assert.containsN(target, ".o-mail-notification-item", 2);
+    assert.containsN(target, ".o-mail-notification-item-name", 2);
+    assert.strictEqual(
+        $(target).find(".o-mail-notification-item-name")[1].innerText.trim(),
+        "Channel"
+    );
+    assert.strictEqual($(target).find(".o-mail-notification-item-counter").text().trim(), "(2)");
+    assert.containsN(target, ".o-mail-notification-item-date", 2);
+    assert.strictEqual($(target).find(".o-mail-notification-item-date")[0].innerText.trim(), "now");
+    assert.strictEqual($(target).find(".o-mail-notification-item-date")[1].innerText.trim(), "now");
+    assert.containsN(target, ".o-mail-notification-item-inlineText", 2);
+    assert.strictEqual(
+        $(target).find(".o-mail-notification-item-inlineText")[1].innerText.trim(),
+        "An error occurred when sending an email"
+    );
+    assert.containsN(target, ".o-mail-notification-item-markAsRead", 2);
+});
+
+QUnit.test("mark as read", async function (assert) {
+    const pyEnv = await startServer();
+    const mailChannelId1 = pyEnv["mail.channel"].create({});
+    const mailMessageId1 = pyEnv["mail.message"].create({
+        message_type: "email",
+        model: "mail.channel",
+        res_id: mailChannelId1,
+        res_model_name: "Channel",
+    });
+    pyEnv["mail.notification"].create({
+        mail_message_id: mailMessageId1,
+        notification_status: "exception",
+        notification_type: "email",
+    });
+    const { click } = await start();
+    await click(".o_menu_systray i[aria-label='Messages']");
+    assert.containsN(target, ".o-mail-notification-item-markAsRead", 2);
+    await afterNextRender(() => {
+        $(target).find(".o-mail-notification-item-markAsRead").click();
+    });
+    assert.containsOnce(target, ".o-mail-notification-item");
+});
+
+QUnit.test("grouped notifications by document", async function (assert) {
```

Comment:
This test is pretty much useless: it's already tested with `mail.channel` above, and showing of chat window is not exactly the intended behavior (should prompt the failure prompt, to mark as read).

Since the behaviour is not implemented, better just drop this test.